feg/gateway/services/n7_n40_proxy/n7/model_conversion.go

```diff
@@ -531,6 +564,26 @@ func GenNotifyUrl(apiRoot string, sessionId string) sbi.Uri {
 	return sbi.Uri(fmt.Sprintf("%s/%s", apiRoot, b64.URLEncoding.EncodeToString([]byte(sessionId))))
 }
 
+func GetSmPolicyUrl(tgppCtx *protos.TgppContext) (string, error) {
+	if tgppCtx == nil {
+		return "", fmt.Errorf("nil TgppCtx")
```

Comment:
try to be consistent and verbose in the errors. 
```
Couldnt get URL from Tgpp Contest: nil TgppCtx"
```

Reply:
Done